mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/StatefulHomeTest.kt

```diff
@@ -286,4 +286,70 @@ class StatefulHomeTest {
     rule.onNodeWithText("test2").assertExists()
     rule.onNodeWithText("test").assertExists()
   }
+
+  @Test
+  fun given_userIsLoggedIn_when_editProfileName_then_nameShouldBeUpdated() = runTest {
+    val auth = buildAuth { user("email@example.org", "password", "1") }
+    val store = buildStore {
+      collection("users") { document("1", ProfileDocument("1", name = "test", emoji = ":)")) }
```

Comment:
You don't need to specify the document id here.

```suggestion
      collection("users") { document("1", ProfileDocument(name = "test", emoji = ":)")) }
```